File: 프로그래머스/lv1/12930. 이상한 문자 만들기/이상한 문자 만들기.js
function solution(s) {
    const arr = s.split(' ');
    let answer = [];
    
    for (let i = 0; i < arr.length; i++) {
        let str = arr[i].split('');
        let res = '';
        
        for (let j in str) {
            if (j % 2 === 0) {
                res += str[j].toUpperCase();
            } else {
                res += str[j].toLowerCase();
            }
        }
        
        answer.push(res);
    }
    
    return answer.join(' ');
}

File: 프로그래머스/1/12930. 이상한 문자 만들기/이상한 문자 만들기.py
def solution(s):
    answer = []
    
    for word in s.split(' '):
        temp = ''
        
        for idx, w in enumerate(word):
            if idx % 2 == 0 :
                temp += w.upper()
            else:
                temp += w.lower()
                
        answer.append(temp)
        
    return ' '.join(answer)